components/atom/button/src/index.js

```diff
@@ -0,0 +1,98 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const CLASS = 'sui-AtomButton'
+const TYPES = ['primary', 'accent', 'secondary', 'tertiary']
+const MODIFIERS = ['disabled', 'small', 'large', 'fullWidth', 'focused']
+
+const appendClass = (suffix) => `${CLASS}--${suffix}`
+const includes = (array, item) => array.indexOf(item) !== -1
+const cleanProps = (props, Component) => {
```

Comment:
This is risky cause the API is fully coupled to button API.
I think it's good so we don't have to declare everything like onClick, accessiblityLabel, value, etc.
On the other hand, if we want out button to be an image someday we can't ensure retrocompatibility.
What are your thoughts about it?

Reply:
I guess that removing propTypes for prod env wouldn't be an option anymore

Reply:
touché !!! :_(